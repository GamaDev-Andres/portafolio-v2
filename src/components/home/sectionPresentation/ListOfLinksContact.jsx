const ListOfLinksContact = () => {
  return (
    <div className="max-w-2xl">
      <span className="uppercase font-medium text-xl text-fondos">
        Hola soy
      </span>
      <h2 className="my-4 uppercase font-bold text-3xl">
        <span className="text-blue-400">Andres</span> Gama
      </h2>
      <p className="whitespace-pre-wrap text-fondos">
        Un
        <span className="text-blue-400 font-medium">
          {' Front-End Web Developer '}
        </span>
        apasionado por el mundo del desarrollo y por crear aplicaciones e
        interfaces que aporten y brinden grandes experiencias.
      </p>
      <div className="py-4 gap-4 flex justify-start items-center flex-wrap">
        <a
          target="_blank"
          className=" text-sm border border-solid shadow-blue-200 shadow-lg border-blue-400 px-6 py-2 rounded-full"
          href="https://drive.google.com/file/d/1IIUutqCGZL_KSN7NqE-JdRYQl_6Qaqhd/view"
          rel="noreferrer"
        >
          Resum√©
        </a>

        <a
          target="_blank"
          className="p-2 flex flex-col justify-center items-center shadow-md shadow-blue-200 border border-solid border-blue-100"
          href="https://github.com/GamaDev-Andres"
          rel="noreferrer"
        >
          <i className="fa-brands fa-github"></i>
          <span className="ml-2 text-xs">Github</span>
        </a>
        <a
          target="_blank"
          className="p-2 flex flex-col justify-center items-center shadow-md shadow-blue-200 border border-solid border-blue-100"
          href="https://www.linkedin.com/in/andres-gama/"
          rel="noreferrer"
        >
          <i className="fa-brands fa-linkedin"></i>
          <span className="ml-2 text-xs">LinkedIn</span>
        </a>
      </div>
    </div>
  )
}

export default ListOfLinksContact
