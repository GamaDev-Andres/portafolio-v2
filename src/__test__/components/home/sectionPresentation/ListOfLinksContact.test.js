/**
 * @jest-environment jsdom
 */
import Enzyme from 'enzyme'
import ListOfLinksContact from '../../../../components/home/sectionPresentation/ListOfLinksContact'

describe('probando <ListOfLinksContact/>', () => {
  const wrapper = Enzyme.mount(
    <ListOfLinksContact />
  )
  test('debe montarse correctamente', () => {
    expect(wrapper).toMatchSnapshot()
    expect(wrapper.find('a').length).toBe(3)
  })
})
