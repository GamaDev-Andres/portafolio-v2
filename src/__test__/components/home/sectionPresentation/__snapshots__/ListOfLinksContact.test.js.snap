// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`probando <ListOfLinksContact/> debe montarse correctamente 1`] = `
<div
  className="py-4 gap-4 flex justify-start items-center flex-wrap"
>
  <a
    className=" text-sm border border-solid shadow-blue-200 shadow-lg border-blue-400 px-6 py-2 rounded-full"
    href="https://drive.google.com/file/d/1IIUutqCGZL_KSN7NqE-JdRYQl_6Qaqhd/view"
    rel="noreferrer"
    target="_blank"
  >
    Resum√©
  </a>
  <a
    className="p-2 flex flex-col justify-center items-center shadow-md shadow-blue-200 border border-solid border-blue-100"
    href="https://github.com/GamaDev-Andres"
    rel="noreferrer"
    target="_blank"
  >
    <i
      className="fa-brands fa-github"
    />
    <span
      className="text-xs"
    >
      Github
    </span>
  </a>
  <a
    className="p-2 flex flex-col justify-center items-center shadow-md shadow-blue-200 border border-solid border-blue-100"
    href="https://www.linkedin.com/in/andres-gama/"
    rel="noreferrer"
    target="_blank"
  >
    <i
      className="fa-brands fa-linkedin"
    />
    <span
      className="text-xs"
    >
      LinkedIn
    </span>
  </a>
</div>
`;
